server/webhook.go

```diff
@@ -123,6 +123,7 @@ func (wh *webhook) PostNotifications(p *Plugin) ([]*model.Post, int, error) {
 			}
 
 			resp, _ := jiraClient.Do(req, nil)
```

Comment:
Doesn't the continue only affect the for loop?  If so, I think you can refactor and share some of code in the if/else blocks.  You could pull out he 

https://github.com/mattermost/mattermost-plugin-jira/blob/d42602692e1602d64a40b75fcc529ffe77cd9c59/server/webhook.go#L120-L130

Reply:
That's my code, I'll make the change and commit. Good catch @jfrerich!

Reply:
Thanks for catching! I was search/replacing and it's really tricky with defer.